WebApp/autoreduce_webapp/reduction_variables/views.py

```diff
@@ -347,19 +347,27 @@ def run_confirmation(request, instrument=None):
         context_dictionary['error'] = 'Runs span multiple experiment numbers (' + ','.join(str(i) for i in rb_number) + ') please select a different range.'
 
     for run_number in run_numbers:
-        old_reduction_run = ReductionRun.objects.filter(run_number=run_number).order_by('-run_version').first()
+        matching_previous_runs_queryset = ReductionRun.objects.filter(run_number=run_number).order_by('-run_version')
```

Comment:
You're not checking the instrument name here so if two instruments have the same run number there will be problems.

Reply:
Good catch